tests/nextflow.config

```diff
@@ -19,20 +19,16 @@ params {
 
 // Impose sensible resource limits for testing
 process {
-    withName: '.*' {
-        cpus   = 2
-        memory = 3.GB
-        time   = 2.h
-    }
-
-    withName: PRESEQ_LCEXTRAP {
```

Comment:
Why is this `PRESEQ_LCEXTRAP` being removed?

Reply:
its now in `conf/base.config`

Reply:
From an already merged PR?

Reply:
Okay found it,

https://github.com/nf-core/methylseq/blob/5bf2730a93c23c46d4572ecd51cfbd0266a0dddc/conf/base.config#L63